src/main/java/com/khadri/jdbc/prepare/data/customer/dao/CustomerInsertDao.java

```diff
@@ -0,0 +1,41 @@
+package com.khadri.jdbc.prepare.data.customer.dao;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Scanner;
+
+import com.khadri.jdbc.prepare.data.customer.dao.model.Customer;
+
+public class CustomerInsertDao {
+	public static void insertData(Customer cust) throws Exception {
+		// write JDBC logic here
+		Scanner sc = new Scanner(System.in);
+		Connection con = null;
+		PreparedStatement pstmt =null;
+
+		try {
+			Class.forName("com.mysql.cj.jdbc.Driver");
```

Comment:
creating connection is common logic for everyone, create a separate class  and prepare connection object in method call that method here to use the connection object.